File: 프로그래머스/unrated/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
    answer = 0
    arr = []
    for i in range(len(rank)):
        if attendance[i]:
            arr.append((rank[i], i))
    s_arr = sorted(arr)[:3]
    
    return s_arr[0][1] * 10000 + s_arr[1][1] * 100 + s_arr[2][1]

File: 프로그래머스/unrated/181851. 전국 대회 선발 고사/전국 대회 선발 고사.cpp
#include <bits/stdc++.h>

using namespace std;

int solution(vector<int> rank, vector<bool> attendance) {
    int answer = 0;
    int n = rank.size();
    vector<pair<int, int>>v;
    for(int i=0;i<n;++i){
        if(attendance[i]) v.push_back({rank[i], i});
    }
    sort(v.begin(), v.end());
    
    int a = v[0].second;
    int b = v[1].second;
    int c = v[2].second;
    
    return 10000 * a + 100 * b + c;
}